lib/const.rb

```diff
@@ -94,3 +94,7 @@
 CREW_CMAKE_OPTIONS = "-DCMAKE_INSTALL_PREFIX=#{CREW_PREFIX} -DCMAKE_LIBRARY_PATH=#{CREW_LIB_PREFIX} -DCMAKE_C_FLAGS=#{CREW_COMMON_FLAGS} -DCMAKE_CXX_FLAGS=#{CREW_COMMON_FLAGS} -DCMAKE_BUILD_TYPE=MinSizeRel"
 CREW_LIB_SUFFIX = if ARCH == 'x86_64' then '64' else '' end
 CREW_CMAKE_LIBSUFFIX_OPTIONS = "-DCMAKE_INSTALL_PREFIX=#{CREW_PREFIX} -DCMAKE_LIBRARY_PATH=#{CREW_LIB_PREFIX} -DLIB_SUFFIX=#{CREW_LIB_SUFFIX} -DCMAKE_C_FLAGS=#{CREW_COMMON_FLAGS} -DCMAKE_CXX_FLAGS=#{CREW_COMMON_FLAGS} -DCMAKE_BUILD_TYPE=MinSizeRel"
+
+PY3_SETUP_BUILD_OPTIONS = "--executable=#{CREW_PREFIX}/bin/python3"
+PY2_SETUP_BUILD_OPTIONS = "--executable=#{CREW_PREFIX}/bin/python2"
+PY_SETUP_INSTALL_OPTIONS = "--root=#{CREW_DEST_DIR} --prefix=#{CREW_PREFIX} -O2 --compile --single-version-externally-managed"
```

Comment:
The `--single-version-externally-managed` option doesn't work for everything.  Try installing aws with it.

Reply:
Could you post an output log @uberhacker?

Reply:
Nevermind I figured it out. It means that aws uses an old version of setup.py. When I port aws to chromebrew without pip, I'll update the setup.py script and patch the changes in the package file. Same with anything else that returns that error.

Reply:
Can we do that now and add to this PR so there is no confusion later?

Reply:
That would require adding so many dependencies and would probably take me a week or so, I don't want to do that right now.

Reply:
That sounds like a bit of an overstatement.  I'll wait but it could save all of us some time knowing that option breaks some builds.